common_rest/src/main/java/uk/gov/dhsc/htbhf/eligibility/model/EligibilityStatus.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.dhsc.htbhf.eligibility.model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import io.swagger.annotations.ApiModel;
+
+/**
+ * The possible values for the eligibility of a claimant.
+ */
+@JsonFormat(shape = JsonFormat.Shape.STRING)
+@ApiModel(description = "The eligibility status")
+public enum EligibilityStatus {
+
+    ELIGIBLE,
+    INELIGIBLE,
+    PENDING,
+    NOMATCH,
```

Comment:
It might be a bit late for this, but as we're changing to use this, could `NOMATCH` become `NO_MATCH`? It's been annoying me for ages now...